build.rs

```diff
@@ -38,7 +39,9 @@ fn main() {
     }
     if lapacke {
         println!("cargo:rustc-link-lib={}=lapacke", kind);
-        println!("cargo:rustc-link-lib={}=tmglib", kind);
+        if tmg {
```

Comment:
I suppose `LAPACKE_WITH_TMG` should be disabled too so that it’s not even compiled.

Reply:
Note also that the features are listed in the readme and in the top-level doc comment. Those lists should be updated as well.

Reply:
Updated.